core/types/rollup_cost.go

```diff
@@ -239,6 +239,7 @@ func NewOperatorCostFunc(config *params.ChainConfig, statedb StateGetter) Operat
 	}
 }
 
+// newOperatorCostFunc returns the operator cost function for Isthmus.
 func newOperatorCostFunc(operatorFeeScalar *big.Int, operatorFeeConstant *big.Int) operatorCostFunc {
```

Comment:
I think `newOperatorCostFuncIsthmus` would be a good name, since we introduce the behavior with Isthmus.